README.md

```diff
@@ -2,8 +2,4 @@
 
 [![Unsupported](https://img.shields.io/badge/pantheon-unsupported-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#unsupported)
 
-Welcome to the edge-integrations-snippets repository! This repository contains example plugins that demonstrate how personalized content can be managed and rendered on a website using Pantheon's Edge Integrations solution.
-
-Each directory is meant to be an individual plugin, and the directory name describes the plugin functionality.
-
-Please ensure you know how to add custom code to your website if you use any of these plugins. These are provided publicly as a courtesy, and are not supported.
\ No newline at end of file
+Welcome to the edge-integrations-snippets repository! This repository contains example plugins that demonstrate how personalized content can be managed and rendered on a website using Pantheon's Edge Integrations solution.
```

Comment:
This might not just have plugins and plugins don't exist in Drupal-speak, so this should be made more generic.

```suggestion
Welcome to the edge-integrations-snippets repository! This repository contains example code, Drupal modules and WordPress plugins that demonstrate how personalized content can be managed and rendered on a website using Pantheon's Edge Integrations solution.
```